website_and_docs/content/blog/2022/selenium-manager-batteries-included-for-selenium-webdriver.md

```diff
@@ -0,0 +1,113 @@
+```
+
+To solve it, you need to download the proper driver ([chromedriver](https://chromedriver.chromium.org/) in this case) and make it available for the test. But doing that manually, the real problem is the driver maintenance since Chrome (and the rest of the major browsers) is _evergreen_, which means that it automatically upgrades itself to the next stable version when available. This browser upgrade will affect the Selenium script since, eventually, a given driver release will no longer be compatible with the local browser. As a result, in the case of Chrome, the Selenium script will fail with the message "_this version of chromedriver only supports chrome version N_" (being N is the latest version of Chrome supported by a particular version of chromedriver). As reported periodically in [StackOverflow](https://stackoverflow.com/search?q=this+version+of+chromedriver+only+supports+Chrome), this is a recurrent problem for Selenium users.
+
+### The driver _managers_
+To solve this problem, several third-party projects of the Selenium ecosystem emerged. For instance, [WebDriverManager](https://bonigarcia.dev/webdrivermanager/) for Java, [webdriver-manager](https://pypi.org/project/webdriver-manager/) for Python, [webdrivers](https://github.com/titusfortner/webdrivers) for Ruby, or [WebDriverManager.Net](https://github.com/rosolko/WebDriverManager.Net) for C#. These tools are sometimes called _managers_ for Selenium WebDriver, and they carry out, to some extent, the driver management process in an automated way.
+
+### Batteries included
+The Selenium project launched the first official [Selenium survey](https://www.selenium.dev/blog/2021/selenium-survey-results/) on January 2021. One of the most evident results of this survey is that most Selenium users want to get rid of the problem of driver management. In other words, Selenium should have some kind of _batteries included_ in it. For this reason, the Selenium project created *Selenium Manager*. This first beta version of Selenium Manager was released on September 2022, and it has been built with the lessons learned from the development of WebDriverManager (first released in 2015).
+
+Selenium Manager is a CLI (Command-Line Interface) tool developed in Rust. You can find its source code on the public GitHub [Selenium repository](https://github.com/SeleniumHQ/selenium/tree/trunk/rust). Moreover, Selenium Manager is built into the CI system of Selenium, i.e., GitHub Actions. If you want to play with it, you can download its development versions (compiled for Windows, Linux, and macOS) from [here](https://github.com/SeleniumHQ/selenium/actions/workflows/build-selenium-manager.yml). Once you have the proper binary of Selenium Manager on your machine, you can execute it from the shell as follows:
```

Comment:
I don't think we should go into this detail in this blog post. We should say why we picked Rust. We should say that storing the binaries in the repo is temporary, but we should point to the binaries there. I'd point people to the rust/README if people are interested in the implementation details rather than discussing them here.